New pages: HTML<form>NoValidate (#36289)
mdn/content: files/en-us/web/api/htmlformelement/novalidate/index.md

---

> title: "HTMLFormElement: noValidate method"
> short-title: noValidate
> slug: Web/API/HTMLFormElement/noValidate
> page-type: web-api-instance-method

Reviewer: ```suggestion
title: "HTMLFormElement: noValidate property"
short-title: noValidate
slug: Web/API/HTMLFormElement/noValidate
page-type: web-api-instance-property
```

browser-compat: api.HTMLFormElement.noValidate
---

{{APIRef("HTML DOM")}}

> The **`noValidate`** property of the {{domxref("HTMLFormElement")}} interface is boolean value indicating if the {{htmlelement("form")}} will bypass [constraint validation](/en-US/docs/Web/HTML/Constraint_validation) when submitted. It reflects the `<form>` element's [`novalidate`](/en-US/docs/Web/HTML/Element/form#novalidate) attribute; if the attribute present, the value is `true`.

Reviewer: ```suggestion
The **`noValidate`** property of the {{domxref("HTMLFormElement")}} interface is a boolean value indicating if the {{htmlelement("form")}} will bypass [constraint validation](/en-US/docs/Web/HTML/Constraint_validation) when submitted. It reflects the `<form>` element's [`novalidate`](/en-US/docs/Web/HTML/Element/form#novalidate) attribute; if the attribute present, the value is `true`.
```

> If this attribute is not set or the value is set to `false`, the form is validated. This can be overridden by setting the {{domxref("HTMLInputElement.formnovalidate")}} and {{domxref("HTMLButtonElement.formnovalidate")}} properties to `true`, or by including `formnovalidate` attributes on {{htmlelement("button")}} or {{htmlelement("button")}} that are used to submit the form.

Reviewer: ```suggestion
If this attribute is not set or the value is set to `false`, the form is validated. This can be overridden by setting the {{domxref("HTMLInputElement.formNoValidate")}} or {{domxref("HTMLButtonElement.formNoValidate")}} property to `true`, either via JavaScript or the HTML `formnovalidate` attribute, for the control used to submit the form.
```
